Plugins/CountWords/CountWordsPlugin.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Linq;
+using System.Text.Json;
+using BasePlugin;
+using BasePlugin.Interfaces;
+using BasePlugin.Records;
+
+
+
+namespace CountWords
+{
+    record Session(int NumberOfWords);
+    public class CountWordsPlugin : IPlugin
+    {
+        public string Id => throw new NotImplementedException();
+
+
+        public bool CanExecute(string input, string session)
+        {
```

Comment:
This function is not needed anymore, look at the IPlugin interface, it was removed